specs/Spec.Hash.Definitions.fst

```diff
@@ -36,33 +36,42 @@ type hash_alg =
   | MD5
   | Blake2S
   | Blake2B
+  | SHA3_256
 
 // BB. Remove when renamed.
 let algorithm = hash_alg
 
+inline_for_extraction noextract
 let is_sha2 = function
   | SHA2_224 | SHA2_256 | SHA2_384 | SHA2_512 -> true
   | _ -> false
 
- inline_for_extraction
+inline_for_extraction noextract
+let is_sha3 = function
+  | SHA3_256 -> true
+  | _ -> false
+
+inline_for_extraction noextract
 let is_blake = function
   | Blake2S | Blake2B -> true
   | _ -> false
 
+inline_for_extraction noextract
 let is_md = function
   | MD5 | SHA1 | SHA2_224 | SHA2_256 | SHA2_384 | SHA2_512 -> true
   | _ -> false
 
 let sha2_alg = a:hash_alg { is_sha2 a }
+let sha3_alg = a:hash_alg { is_sha3 a }
 let blake_alg = a:hash_alg { is_blake a }
 let md_alg = a:hash_alg { is_md a }
 
- inline_for_extraction
+inline_for_extraction
```

Comment:
Why no `noextract` here? Same for following definitions. This predates this PR, but it would be good to be consistent throughout the definitions in this module (this is clearly non-blocking given the current size of this PR)

Reply:
I generally don't bother with noextract in specs because all of those are eliminated anyhow via `-bundle` in the final krml invocation